### ISSUE TITLE
Use of eval instead of getattr

### ISSUE THREAD

**Issue by jenner** [xuru/pyvisdk#26]
Hey guys,

just wondering -- what's the reason behind usage of eval in pyvisdk/core.py#253 ? I mean this piece of code:

``` python
    def _create_do_obj(self, class_name, obj_content, parent):
        mod_name = 'pyvisdk.do.%s' % camel_to_under(class_name)
        do = import_string('%s.%s' % (mod_name, class_name))

        kwargs = {}
        for attr_name in filter(lambda x: not x.startswith('_'), dir(obj_content)):
            # evil eval
            attr_data = eval('obj_content.%s' % attr_name)
            kwargs[attr_name] = self._parse_object_content(attr_data, parent=do)

        rv = do(self, **kwargs)
        return rv
```

It seems like you could just use getattr() here.

I'm asking because I've got a SyntaxError recently because one of the returned types contained python keywords in it (the "from" property of an Alarm object or something). My workaround was to replace the eval with

```
attr_data = getattr(obj_content, attr_name)
```

**Comment by xuru** [comment]
Only inexperience at the time.  Please submit a pull request, and I'll make the change.  Unfortunetelly, I don't have an ESX to test against, so I won't be able to verify before accepting, but this is small enough, that I think we're fine.

**Comment by xuru** [comment]
Closed with pull request #27.  Thanks for the help!
